docs/rules/indent.md

```diff
@@ -0,0 +1,82 @@
+# Validate Indentation (indent)
+
+This option validates a specific tab width for your code in block statements.
```

Comment:
This section should talk about indentation and indentation styles. You should why indentation is used and what the various styles are.